src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  if (typeof numberOne !== 'number' || typeof numberTwo !== 'number' || !operation) return '0';
+  const nOne = Big(Number(numberOne));
+  const nTwo = Big(Number(numberTwo));
+  let r = 0;
```

Comment:
### [Optional]
For good naming conversion, it would be nice to make use of `result` instead of `r` so that the naming can be related to the logic, which is to get the result of the calculation.